src/components/TabSelector/getOpacity/types.ts

```diff
@@ -0,0 +1,14 @@
+import type {Animated} from 'react-native';
+
+type GetOpacityConfig = {
+    routesLength: number;
+    tabIndex: number;
+    active: boolean;
+    affectedTabs: number[];
+    position?: Animated.AnimatedInterpolation<number>;
+    isActive?: boolean;
+};
+
+type GetOpacity = (args: GetOpacityConfig) => 1 | 0 | Animated.AnimatedInterpolation<number> | undefined;
```

Comment:
I don't see an undefined return.
```suggestion
type GetOpacity = (args: GetOpacityConfig) => 1 | 0 | Animated.AnimatedInterpolation<number>;
```